### PR TITLE
Fix golangci-lint invocation

### PR DESCRIPTION
**Pull request by dirkmueller**
The old version didn't actually run with go 1.13 anymore, so it
just exited right away. update to a working version and fix the fallout.

## Why is this PR needed?

Does it fix an issue? addresses a business case?

add a description and link to the issue if one exists.

Fixes #

**Reminder**: Add the "fixes bsc#XXXX" to the title of the commit so that it will
appear in the changelog.

## What does this PR do?

please include a brief "management" technical overview (details are in the code)

## Anything else a reviewer needs to know?

Special test cases, manual steps, links to resources or anything else that could be helpful to the reviewer.

## Info for QA

This is info for QA so that they can validate this. This is **mandatory** if this PR fixes a bug.
If this is a new feature, a good description in "What does this PR do" may be enough.

### Related info

Info that can be relevant for QA:
* link to other PRs that should be merged together
* link to packages that should be released together
* upstream issues

### Status **BEFORE** applying the patch

How can we reproduce the issue? How can we see this issue? Please provide the steps and the prove
this issue is not fixed.

### Status **AFTER** applying the patch

How can we validate this issue is fixed? Please provide the steps and the prove this issue is fixed.

## Docs

If docs need to be updated, please add a link to a PR to https://github.com/SUSE/doc-caasp.
At the time of creating the issue, this PR can be work in progress (set its title to [WIP]),
but the documentation needs to be finalized before the PR can be merged. 

# Merge restrictions

(Please do not edit this)

We are in *v4-maintenance phase*, so we will restrict what can be merged to prevent unexpected surprises:

    What can be merged (merge criteria):
        2 approvals:
            1 developer: code is fine
            1 QA: QA is fine
        there is a PR for updating documentation (or a statement that this is not needed)

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
Tests pass and I do not forsee anything re CI that could go wrong, so I am accepting it. When you have the other approval, feel free to  merge.

File: Makefile [hunk 1]
```diff
@@ -98,8 +98,8 @@ lint: deps
 
 .PHONY: deps
 deps:
-	test -f $(BINPATH)/golangci-lint || curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(BINPATH) v1.21.0
-	test -f $(BINPATH)/shellcheck || curl -sfL "https://storage.googleapis.com/shellcheck/shellcheck-v0.4.7.linux.x86_64.tar.xz" | tar -xJv --strip-components=1 -C $(BINPATH)
+	test -f $(BINPATH)/golangci-lint || curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.27.0/install.sh | sh -s -- -b $(BINPATH) v1.27.0
+	test -f $(BINPATH)/shellcheck || curl -sfL "https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz" | tar -xJv --strip-components=1 -C $(BINPATH)
```
Comment (Itxaka) [inline Makefile:102]: this is not mentioned in the commit message, was this supposed to be a part of the same commit?

Comment (dirkmueller) [reply Makefile:102]: yeah, it was intentional. I can split it out into a separate PR if you prefer.

Comment (Itxaka) [reply Makefile:102]: just an updated commit message would be good enough for me and my text search on git history :p

Comment (dirkmueller) [reply Makefile:102]: I splitted it into a separate commit with a separate commit message under the same PR.

**Pull request merged**